client/src/pages/Home.jsx

```diff
@@ -75,21 +76,23 @@ export default function Home() {
         </DialogActions>
     </Dialog>
 
-    <Grid container justifyContent="center" className={classes.MarginTop}>
-      <Grid item sm={6} xs={12}>
-        <Typography variant="subtitle">Code for Chicago</Typography>
+    <RegPage container justifyContent="center" >
```

Comment:
There is a component we already created that you can use instead of this - `Card`, in `styles/components/Card.style'. If you import that and use it you don't need to re-create the styles